### PR TITLE
[AttributeBundle] Form registration, missing specs and CS

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/AttributeBundle/spec/DependencyInjection/SyliusAttributeExtensionSpec.php [hunk 1]
```diff
@@ -42,63 +42,63 @@ function it_processes_the_configuration_and_registers_services_per_subject(Conta
...
+//                '%sylius.validation_group.product_attribute_value%',
+//                'product'
+//            ))
+//        ;
+//
+//        $attributeValueFormType
+//            ->addTag('form.type', array('alias' => 'sylius_product_attribute_value'))
+//        ;
+
+//        $container->setDefinition('sylius.form.type.product_attribute_value', $attributeValueFormType)->shouldBeCalled();
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/AttributeBundle/spec/DependencyInjection/SyliusAttributeExtensionSpec.php]: ?

Comment (aRn0D) [inline src/Sylius/Bundle/AttributeBundle/spec/DependencyInjection/SyliusAttributeExtensionSpec.php]: I forget to remove it !

**Comment by aRn0D** [comment]
@pjedrzejewski I delete the comments.

**Comment by pjedrzejewski** [comment]
Thank you Mr. Arnaud! :+1:

**Pull request merged**